doc/radosgw/lua-scripting.rst

```diff
@@ -57,6 +57,13 @@ To add a package to the allowlist:
   # radosgw-admin script-package add --package={package name} [--allow-compilation]
 
 
+To add a specific version of a package to the allowlist:
```

Comment:
please add a note on adding a version when another version (or the same package without a version) already exists